application/common/tests/test_utilities.py

```diff
@@ -104,17 +106,32 @@ def test_is_val_none_or_empty(val, expected):
     assert is_val_none_or_empty(val) == expected
 
 
-@mark.parametrize("input_dict,keys_tobe_removed,msg_limit,expected", [
-    ({"Name": "John", "Address": ["2", "4"], "Age": 34}, ["Address"], 20, {"Name": "John", "Age": 34}),
-    ({"Name": "John", "Address": ["2", "4"], "Age": 34}, ["Address", "Age"], 20, {"Name": "John"}),
-    ({"Name": "John", "Address": ["2", "4"], "Age": 34}, [""], 20, {"Name": "John", "Address": ["2", "4"], "Age": 34}),
-    ({"Name": "John", "Age": 34}, ["Age"], 120, {"Name": "John", "Age": 34})
-    ])
+@mark.parametrize(
```

Comment:
Is there a reason why the formatting has changed to expanded this?

Reply:
Automatic black formatting has caused this and therefore can't be removed